lib/use-cases/updateArchiveStatus.js

```diff
@@ -0,0 +1,16 @@
+module.exports = ({ gateways: { dropbox: dropboxes } }) => {
```

Comment:
Maybe rename this file to match the others - so `UpdateArchiveStatus` vs `updateArchiveStatus`